### ISSUE TITLE
Redesign: Associate customer actions to it's case

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#18864]
### During UAT stakeholders expressed confusion with the customer actions feature, and asked us to display the actions with the project vs in it's own space. 

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTJnQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e42bf82b5fef98e5bff4c6a8bb446ab7145f1f38/image.png" alt="image.png" />

**Comment by ChrispinP** [comment]
Customer Actions have been updated so they are now a 1-1 relationship with each case. Each action has an associated Type, Text, & Status that now update base on records rules along the customer's journey. Every action also has a Triggered Date of when the "switch" was flipped.

Since there is now only one action per case, only upon case creation is the action record created. Thereafter, the action record is updated and overwritten with the newest action.

Still need to update all the rules and determine best way to display/notify customer

**Comment by ChrispinP** [comment]
During refinement of the project/request creation flow, Karo & I determined that all customer action items are now addressed either by the flow or page messages. Therefore, Customer actions are irrelevant and no longer needed. Based on testing, we believe we have added sufficient guardrails to help the user and utilizing customer actions with a non-existant notification system would provide trivial additional value. We are removing customer actions as a feature.
